ci/setup.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash -eu
 
-if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+if uname -s | grep -q Linux; then
   sudo modprobe fuse
```

Comment:
Mmmh yes according to the Circle build log, some kernel files are missing.
Re-installating (`apt-get`) the kernel or reconfiguring it (`dpkg-reconfigure`) could help, but these are quite uncommon build steps :)